x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/method_connector/method_connector.tsx

```diff
@@ -43,10 +46,18 @@ export const MethodConnector: React.FC<{ isNative: boolean }> = ({ isNative }) =
   const { isModalVisible } = useValues(AddConnectorLogic);
   const { setIsModalVisible } = useActions(AddConnectorLogic);
   const { fullIndexName, language } = useValues(NewSearchIndexLogic);
-  const { isCloud } = useValues(KibanaLogic);
+  const { isCloud, cloud } = useValues(KibanaLogic);
   const { hasPlatinumLicense } = useValues(LicensingLogic);
+  const { data } = useValues(FetchCloudHealthApiLogic);
 
   const isGated = isNative && !isCloud && !hasPlatinumLicense;
+  const hasLowMemory = isNative && isCloud && data && !data.has_min_connector_memory;
+
+  const { makeRequest: fetchCloudHealth } = useActions(FetchCloudHealthApiLogic);
+
+  useEffect(() => {
+    fetchCloudHealth({});
```

Comment:
Should we still fetch this when `!isCloud`?

Reply:
Probably not! Fixed :)